### ISSUE TITLE
[COGNITION] Save in progress dialog should be above the form navigation

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#5953]
## Description

This issue was originally brought up with a Higher-Level Review a11y review ([ref](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3566#issuecomment-559214317)), but is an issue associated with all forms. cc @jenstrickland

After interacting any form element, the "Application has been saved" message will appear below the form "Back" and/or "Continue" buttons. This may be of more use to screenreader users if this content was before the form buttons.

![](https://user-images.githubusercontent.com/57469/69751582-6d146b80-111d-11ea-90f8-f1af30e9bbd8.png)

## Acceptance Criteria

- [ ] Ensure the screenreader reads the application has been saved message
- [ ] Move the save message box above the form controls

## Steps to Recreate

1. Log in as user 218, then go to a form that contains the informal conference questions, e.g. https://staging.va.gov/disability/file-disability-claim-form-21-526ez/alternate-names.
2. Choose either radio button option.
3. The "Application has been saved" message will appear _below_ the "Back" and "Continue" buttons.
    ![Screen Shot 2020-02-18 at 1 22 58 PM](https://user-images.githubusercontent.com/136959/74770448-e40b5d00-5251-11ea-938c-393d4800c8f8.png)

**Comment by 1Copenut** [comment]
Please see this design issue for related context: https://github.com/department-of-veterans-affairs/va.gov-team/issues/182

/cc @jenstrickland

**Comment by Mottie** [comment]
Oops, I think this is probably a duplicate then. Do you want to close this one?

**Comment by Mottie** [comment]
Closing, as it is a duplicate issue.
